src/app/core-ui/material/mat-select-search/mat-select-search.component.ts

```diff
@@ -59,15 +68,20 @@ export class MatSelectSearchComponent implements OnInit, OnChanges {
     return this.options.filter(option => option[this.showValueOf].toLowerCase().includes(filterValue));
   }
 
-  public _onBlur($event: any): void {
-    const val = $event.target.value;
+  public _onBlur(): void {
```

Comment:
Why was $event not working?

Reply:
yes, that's working fine.
But in the same component, we have the ``Viewchild`` (native element) ref of the <input-field>.
that's the reason I'm replaced ;

``` $event.target = this.textInput.nativeElement  ```